queries.sql

```diff
@@ -10,4 +10,12 @@ SELECT date_of_birth FROM animals WHERE (name ='Agumon') OR (name = 'Pikachu');
 SELECT name, escape_attempt FROM animals WHERE (weight_kg < 10.5);   
 SELECT * FROM animals WHERE (neutered = 't');    
 SELECT * FROM animals WHERE (name != 'Gabumon');   
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;   
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;   
+
```

Comment:
- [ ] It would be better if we could find the following queries in the queries.sql file
![Screenshot from 2022-08-10 14-12-55](https://user-images.githubusercontent.com/46388113/183923667-9c2364f6-39a9-4552-bae4-13ee3b02cf4a.png)

![Screenshot from 2022-08-10 14-14-26](https://user-images.githubusercontent.com/46388113/183924062-5eaaa208-a575-4903-88d7-5d6c2938aa69.png)

![Screenshot from 2022-08-10 14-16-18](https://user-images.githubusercontent.com/46388113/183924479-4b59f8da-9db4-422d-9829-698cf732d84b.png)

### NB:
I HAVE LOOKED AT THE QUERIES BUT I'M UNABLE TO FIND THESE QUERIES IN THE QUERIES.SQL FILE, HOWEVER I DO FIND THE SCREENSHOT WITH THOSE QUERIES. KINDLY MOVE THEM INTO THE QUERIES FILE